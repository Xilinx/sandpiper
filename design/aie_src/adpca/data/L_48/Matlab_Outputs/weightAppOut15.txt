-5.341715813e+00 -4.750635147e+00
-2.441391945e-01 1.381291771e+01
3.681145668e+00 -3.405954599e+00
