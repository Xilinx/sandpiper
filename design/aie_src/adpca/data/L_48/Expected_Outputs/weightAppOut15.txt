-5.341455460e+00 -4.750617981e+00 
-2.441908121e-01 1.381251049e+01 
3.681117773e+00 -3.405890226e+00 
